docs/source/deploying/prefixed-root.md

```diff
@@ -0,0 +1,62 @@
+---
+html_meta:
+  "description": "Prefix (non-root) deployment for Volto"
+  "property=og:description": "Prefix (non-root) deployment for Volto"
+  "property=og:title": "Prefixed (non-root) deployment"
+  "keywords": "Volto, deployment"
+---
+
+# Prefixed (non-root) deployment
+
+If you're integrating a Volto website with another existing website, you may
+need to run Volto on a "folder" inside that website, rather than
+Volto directly serving the root folder.
+
+The first step is to set an environment variable, `RAZZLE_PREFIX_PATH` to the
+path "prefixed path" of your Volto. For example, if I want Volto's root to be
+hosted at `http://example.com/my-prefix`, you need to start Volto with:
+
+```
+RAZZLE_PREFIX_PATH=/my-prefix yarn start
+```
+
+If you need to debug and understand how the requests are rewritten by the Volto
+SSR, you can add the following environment variables to the Volto start line:
+
+
+```
+DEBUG_HPM=true DEBUG=superagent RAZZLE_PREFIX_PATH=/my-prefix yarn start
+```
+
+The prefix location will be used regardless of how you start Volto, in
+development or production mode. When developing, though, if your backend is
+something other then `http://localhost:8080`, you'll need to provide your own
+solution on how to handle things.
+
+For production setup, when hosting Volto behind a proxy HTTP server, you can
+configure your rewrite rules to something like this (in this case, for Apache):
+
+```apache
```

Comment:
We didn't settle on Apache or Nginx syntax for examples on web server side implementations. Do you think that it's confusing not settling on one? Examples are mixed now all over the place. It's ok for me, but for users could be confusing. Either I'll include both, or settle on one. What do you think? @stevepiercy ?

Reply:
I think we do not need to settle on one server. We could [display multiple examples in tabs](https://sphinx-design.readthedocs.io/en/furo-theme/tabs.html#tabbed-code-examples). If we currently have examples for more than one server, then let's use that. If we must work to come up with more than one example, then let's go with what we have now.